src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -0,0 +1,410 @@
+                    top_p=hyperparams.get("top_p", 0.9)
+                )
+                
+                response = await llm_manager.generate(request)
+                self.total_llm_cost = getattr(self, 'total_llm_cost', 0.0) + response.cost
+                
+                # Extract the core question
+                content = response.content.strip()
+                match = re.search(r'Q:\s*(.+)', content)
+                if match:
+                    return match.group(1).strip()
+                else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)